puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/version_map.groovy

```diff
@@ -0,0 +1,5 @@
+def versions = [
```

Comment:
We have a similar mapping in the katello job. Can we unify that in 1 file so we can't forget one?